src/components/Score.js

```diff
@@ -3,16 +3,40 @@ import { browserHistory } from 'react-router';
 
 import ScoreInput from './ScoreInput.js';
 
+const pushPayload = {
+  sessionId: null,
```

Comment:
What do you think about putting thing inside a `score.js` and creating a factory function around it. That object could also have a function that saves it?